### PR TITLE
Add KeyGestureRecognizer classification tests

### PR DESCRIPTION
**Pull request by cl445**
## What

Unit tests for the pure classification logic of `KeyGestureRecognizer`, which was barely covered so far (`angleToGestureType` 0 %, `classify(features:)` ~29 %).

## Covered

- **`angleToGestureType`** — all eight swipe sectors, normalization of negative angles (atan2 convention) and the wraparound near 360°
- **`classify(features:)`** — tap, circular-CW/CCW and directional (swipe/return) branches via synthetic `GestureFeatures` vectors

The entire unit-testable classification surface is thus fully covered.

## Not covered (deliberately)

The `DragGesture` closures in `body(content:)` require UI-level interaction and remain reserved for UI tests.

## Test run

8 new tests green on `iPhone 17 Pro`.

## Summary by CodeRabbit

* **Tests**
  * Added coverage for gesture classification to improve confidence in touch/gesture behavior.
  * Verified handling of edge cases such as angle wrap-around, directional swipes, tap detection, circular gestures, and return-swipe recognition.

### CODE REVIEW

**Comment by cl445** [comment]
@coderabbitai full review

**Comment by cl445** [comment]
@coderabbitai full review

File: wurstfingerTests/KeyGestureClassificationTests.swift [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        thresholds: GestureClassificationThresholds = .default,
+        maxDisplacement: CGFloat = 100,
+        maxDisplacementProgress: CGFloat = 1.0,
+        returnRatio: CGFloat = 1.0,
+        maxDisplacementAngle: CGFloat = 0,
+        angularSpan: CGFloat = 0,
+        circularity: CGFloat = 0,
+        turnConsistency: CGFloat = 0,
+        orientedCompactness: CGFloat = 0,
+        pathLength: CGFloat = 100
```
[the remark this answers is not in the document]
Comment (cl445) [reply wurstfingerTests/KeyGestureClassificationTests.swift:45]: Übernommen: Fixture nutzt jetzt test-lokale, gepinnte `GestureClassificationThresholds` statt `.default`, und die Test-Inputs werden daraus abgeleitet (z.B. `minSwipeLength/2`, `minAngularSpan + 0.5`). Damit kann Produktions-Retuning keine Branch-Grenze still verschieben.
